resources/scripts/components/server/settings/SettingsContainer.tsx

```diff
@@ -50,7 +54,10 @@ export default () => {
                                     </div>
                                 </div>
                                 <div css={tw`ml-4`}>
-                                    <a href={`sftp://${username}.${id}@${ip(sftp.ip)}:${sftp.port}`}>
+                                    <a
+                                        href={`sftp://${username}.${id}@${sftp.alias ? sftp.alias : ip(sftp.ip)}:${sftp.port
+                                            }`}
```

Comment:
I think this needs to be one 1 line 
```suggestion
                                        href={`sftp://${username}.${id}@${sftp.alias ? sftp.alias : ip(sftp.ip)}:${sftp.port}`}
```

Reply:
It's fine. eslint formatted it this way.